js/common/lib/tensor.ts

```diff
@@ -234,5 +234,71 @@ export interface TensorConstructor {
   // #endregion
 }
 
+export interface ImageFromTensorOptions {
```

Comment:
`TensorToImageOptions` is a better name because it is the same way to how the function is named.
```ts
Tensor.fromImage(...)  // options are TensorFromImageOptions

myTensor.toImage(...) // options are TensorToImageOptions
```

Reply:
Agreed this is a better name. Update the code